src/common/com/intellij/plugins/haxe/ide/HaxeCompilerCompletionContributor.java

```diff
@@ -120,16 +123,24 @@ protected void addCompletions(@NotNull CompletionParameters parameters,
                        //Export/flash/haxe contains build.hxml which gets generated after build
                        break;
                      case HaxeModuleSettingsBaseImpl.USE_OPENFL:
-                       commandLineArguments.add(HaxelibCommandUtils.getHaxelibPath(moduleForFile));
-                       commandLineArguments.add("run");
-                       commandLineArguments.add("lime");
-                       commandLineArguments.add("display");
-                       //flash, html5, linux, etc
                        String targetFlag = moduleSettings.getNmeTarget().getTargetFlag();
-                       commandLineArguments.add(targetFlag);
 
-                       List<String> stdout = HaxelibCommandUtils.getProcessStdout(commandLineArguments,
-                                                                                  BuildProperties.getProjectBaseDir(project));
+                       List<String> stdout = openFLDisplayArguments.get(moduleForFile.getModuleFilePath() + targetFlag);
+                       if (stdout == null) {
```

Comment:
It's only affects `lime display {target}`, which is used to get called every single time, and seems like there are some performance problems, I think I had similar issue with HIDE.